group_vars/infrastructure.yml

```diff
@@ -83,5 +83,9 @@ poi_postgres_password: !vault |
   6539316338353964333762366332383338626532616137653036
 
 poi_domain: "features.stadtnavi.eu"
+api_hostname: "api.dev.stadtnavi.eu"
 
-api_hostname: "api.dev.stadtnavi.eu"
\ No newline at end of file
+mqtt_domain: "vehiclepositions.stadtnavi.eu"
+mqtt_otp_domain: "api.dev.stadtnavi.eu"
```

Comment:
What is the mqtt_otp_domain required for? What is necessary, one we deploy to prod?
Could you provide some documentation regarding the mqtt setup, please?